src/filters/filter-results.html

```diff
@@ -12,7 +12,7 @@
         </li>
       </ul>
       <p><a class="clear-filters" ng-click="$ctrl.clearAllFilters()" ng-if="$ctrl.config.appliedFilters.length > 0">Clear All Filters</a></p>
-      <div ng-if="$ctrl.config.totalCount !== 0" class="pf-table-view-selected-label">
+      <div ng-if="$ctrl.config.selectedCount !== undefined && $ctrl.config.totalCount !== undefined" class="pf-table-view-selected-label">
         <strong>{{$ctrl.config.selectedCount}}</strong> of <strong>{{$ctrl.config.totalCount}}</strong> selected
```

Comment:
@dtaylor113 would it make more sense to add the tests `angular-patternflytest/filters/filter.spec.js`? (sure as crap is easier 😆 )